nix/hetzner-bootstrap.nix

```diff
@@ -5,7 +5,6 @@ let
 
   nixpart = python2Packages.nixpart0.override {
     useNixUdev = false;
-    udevSoMajor = 0;
```

Comment:
Ah, that's actually something I wanted to break up into a separate commit, but it was left in that stash.

Reply:
Done